src/app/debug/filter-side-drawer/filter-side-drawer.component.css

```
@@ -0,0 +1,59 @@
+.filter-vertical-panel {
+  position: fixed;
+  top: 0;
+  right: 0;
+  height: 100%;
+  width: 300px;
+  background-color: #f0f0f0;
+  box-shadow: 0 0 8px 1px gray;
+  z-index: 1000;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.filter-input-container {
+  margin-left: 1rem;
+  margin-right: 1rem;
```

This can be shortened:
```suggestion
  margin: 0 1rem;
```